go.mod

```diff
@@ -71,6 +71,7 @@ replace (
 	github.com/containers/image => github.com/openshift/containers-image v0.0.0-20190130162819-76de87591e9d
 	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20191121165722-d1d5f6476656
 	github.com/onsi/ginkgo => github.com/openshift/onsi-ginkgo v4.7.0-origin.0+incompatible
+	github.com/redhat-developer/service-binding-operator => github.com/kadel/service-binding-operator v0.4.1-0.20211207095808-c49634ebeb6f
```

Comment:
TODO (@kadel): his was done to verify fix for panic. Fix was merged upstream (https://github.com/redhat-developer/service-binding-operator/pull/1079) Now this is not needed, and we should be able to  use upstream service-biding-operator

Reply:
done